graphics/nanovg/Portfile

```diff
@@ -11,9 +11,10 @@ maintainers         @jasonliu-- openmaintainer
 
 if {$subport eq "nanovg"} {
 
-github.setup        memononen $subport 2bead03bea43b2418060aaa154f972829995e663
-version             20200320
+github.setup        wjakob $subport bf2320d1175122374a9b806d91e9e666c9336375
+version             20190524
 revision            0
+epoch               1
```

Comment:
this is just a comment FYI, but note that once you bump the epoch, it can never go down again for a given port. Not a problem, just a note to not to remove it on a subsequent update.

Reply:
Wow, yeah it says that in the MacPorts guide too. I'll keep that in mind.